src/main/java/org/ibp/api/java/impl/middleware/study/validator/StudyEntryValidator.java

```diff
@@ -94,14 +95,12 @@ public void validateStudyAlreadyHasStudyEntries(final Integer studyId) {
 
 	}
 
-	public void validateStudyEntryProperty(final Integer studyEntryPropertyDataId) {
+	public void validateStudyEntriesForUpdate(final List<Integer> entryIds) {
 
-		this.errors = new MapBindingResult(new HashMap<String, String>(), Integer.class.getName());
-
-		final Optional<StudyEntryPropertyData> studyEntryPropertyData =
-			this.middlewareStudyEntryService.getStudyEntryPropertyData(studyEntryPropertyDataId);
-		if (!studyEntryPropertyData.isPresent()) {
-			this.errors.reject("invalid.study.entry.property.data.id");
+		this.errors = new MapBindingResult(new HashMap<String, String>(), String.class.getName());
```

Comment:
This should be redesigned as `validateStudyEntriesUsedInPlots(final List<Integer> entryIds, boolean shouldBeUnused)`.
..
```
if  (!CollectionUtils.isEmpty(entriesWithPlot) && shouldbeUnused) {
  // then raise error
}
```